go-controller/pkg/ovn/controller/apbroute/external_controller_namespace_test.go

```diff
@@ -192,23 +192,24 @@ var _ = Describe("OVN External Gateway namespace", func() {
 				}, 5).Should(Equal(staticPolicy.Spec))
 				Eventually(func() []string { return listNamespaceInfo() }, 5).Should(HaveLen(1))
 				Eventually(func() *namespaceInfo { return getNamespaceInfo(namespaceTest.Name) }, 5).Should(
-					BeEquivalentTo(
+					BeComparableTo(
 						&namespaceInfo{
-							policies:        sets.New(staticPolicy.Name),
-							staticGateways:  gatewayInfoList{{gws: sets.New(staticHopGWIP)}},
-							dynamicGateways: make(map[ktypes.NamespacedName]*gatewayInfo)}))
+							Policies:        sets.New(staticPolicy.Name),
+							StaticGateways:  gatewayInfoList{newGatewayInfo(sets.New(staticHopGWIP), false)},
+							DynamicGateways: make(map[ktypes.NamespacedName]*gatewayInfo)}, cmpOpts...))
```

Comment:
nit: `cmdOpts`in a new line for readibility

Reply:
done